app/views/smart_answers/inputs/_country_select_question.html.erb

```diff
@@ -1,4 +1,4 @@
-<span class="govuk-caption-l"><%= question.caption %></span>
+<%= render "smart_answers/inputs/caption", { question: question } %>
```

Comment:
I don't think you need the curly brackets in these render calls. This should work:
```ruby
<%= render "smart_answers/inputs/caption", question: question %>
```

Reply:
Thanks! Removed the extra brackets around these.